src/algebra/continued_fractions/computation/approximations.lean

```diff
@@ -135,18 +145,15 @@ lemma of_part_num_eq_one_and_exists_int_part_denom_eq {gp : gcf.pair K}
   (nth_s_eq : (gcf.of v).s.nth n = some gp) :
   gp.a = 1 ∧ ∃ (z : ℤ), gp.b = (z : K) :=
 begin
-  obtain ⟨ifp, stream_succ_nth_eq, ifp_b_eq_gp_b⟩ :
-    ∃ ifp, int_fract_pair.stream v (n + 1) = some ifp ∧ (ifp.b : K) = gp.b,
+  obtain ⟨ifp, stream_succ_nth_eq, _⟩ :
```

Comment:
```suggestion
  obtain ⟨ifp, stream_succ_nth_eq, -⟩ :
```
If you want to throw away the variable, use `-`, with `_` it get's auto-named. (I'm assuming you wanted to discard the variable, otherwise ignore the suggestion.)

Reply:
Changed it were applicable here and other places in `continued_fractions`.